chapter_5/exercise_5_10/expr_test.sh

```
@@ -0,0 +1,87 @@
+# Test 5: Division by zero
+T=5
+result=$(./expr 5 0 / 2>&1)
+if [[ "$result" == *"division by zero"* ]]; then
+    echo "Test $T PASS: Division by zero detected"
+else
+    echo "Test $T FAIL: Should detect division by zero"
+fi
+
+# Test 6: Missing operator
+T=6
+result=$(./expr 5 1)
```

Shellcheck (0.11.0)

[warning] 'expr' expects 3+ arguments, but sees 2. Make sure each operator/operand is a separate argument, and escape <>&|.

(SC2307)